install/helm/openchoreo-control-plane/Chart.yaml

```diff
@@ -28,3 +28,7 @@ dependencies:
     repository: oci://quay.io/jetstack/charts
     condition: cert-manager.enabled
     version: "v1.16.2"
+  - name: traefik
+    repository: https://traefik.github.io/charts
+    condition: traefik.enabled
+    version: "34.4.1"
```

Comment:
isn't this conflicts with the traefik installed by default with k3d cluster? So don't we need to disable this in our single cluster/multi-cluster/quick-start values?